config/filter.d/mysqld-auth.conf

```diff
@@ -17,7 +17,7 @@ before = common.conf
 
 _daemon = mysqld
 
-failregex = ^%(__prefix_line)s(?:\d+ |\d{6} \s?\d{1,2}:\d{2}:\d{2} )?\[Warning\] Access denied for user '\w+'@'<HOST>' (to database '[^']*'|\(using password: (YES|NO)\))*\s*$
+failregex = ^%(__prefix_line)s(?:\d+ |\d{6} \s?\d{1,2}:\d{2}:\d{2} )?\[\w+\] Access denied for user '[^']+'@'<HOST>' 
 
```

Comment:
Oh yeah, forget to mention: to exclude wrong failures, I've of course taken a look for similar error in mysql (currently only this 2 messages):
https://github.com/mysql-inception/inception/search?utf8=%E2%9C%93&q=%22Access+denied+for+user%22&type=Code